docs/source/release_notes.rst

```diff
@@ -2,8 +2,18 @@
 
 Release Notes
 -------------
-Future Release
-==============
+.. Future Release
+    ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v0.5.0 July 7, 2021
```

Comment:
Super minor point, but we have been using 3-digit month abbreviations as it displays better in the documentation side bar. `July` -> `Jul`